lib/Test2/Event/ParseError.pm

```diff
@@ -0,0 +1,20 @@
+package Test2::Event::ParseError;
+use strict;
+use warnings;
+
+our $VERSION = '0.000012';
+
+BEGIN { require Test2::Event; our @ISA = qw(Test2::Event) }
```

Comment:
Test2::Harness does not have the strict performance requirement of the Test2 dist, we can just use base/parent in events that should not move to Test2 itself.